source/application/AVIDumping/AVIDumper.cpp

```diff
@@ -6,9 +6,9 @@
 
 #include <windows.h>
 
-#include "AVIDumper.h"
```

Comment:
include sorting failed in this file, see if you can make it more aggressive?

Reply:
As far as I can see it respects the manual blank lines as separators. Not sure if I can do something about that, I'll look.

Reply:
Doesn't look like I can do something about this (apart from manually getting rid of the blank lines).

Reply:
Then it will be  a task for another PR.